README.md

```diff
@@ -34,6 +40,18 @@ To streamline the integration with user interfaces and data management systems (
 RTC-Tools uses [CasADi](https://web.casadi.org/) as a symbolic framework for algorithmic differentiation, as well as for interfacing with numerical optimization solvers.
 
 
+## Migration from GitLab
```

Comment:
I suggest moving the Migration from GitLab section to follow the Documentation section. This change is recommended because the Migration from GitLab section is quite specific, and it is already referenced in line 11